### PR TITLE
Adjust desktop sync clients and mobile apps headings in system_requirements

### PR DESCRIPTION
**Pull request by phil-davis**
Here is my suggestion to sort out the desktop and mobile headings in `system_requirements.adoc`

There is a "Desktop Sync Client" and there are "Mobile Apps". The "Desktop Sync Client" can run on a range of operating systems, which are listed in that section. The "Mobile Apps" are available for/on the operating systems mentioned in that section.

Maybe we also want to re-order the sections "Desktop Sync Client", "Web Browser" and "Mobile Apps"? (I did not do that in this PR because then the diff listing will be obscured by large cut-paste, and I actually I don't know which order people prefer).

Please comment, and we can sort this out then backport it into 10.4 and 10.3

### CODE REVIEW

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 1]
```diff
@@ -73,7 +73,9 @@ For Linux distributions, we support, if technically feasible, the latest two ver
 * Xen
 * KVM
 
-=== Desktop
+=== Desktop Sync Client
+
+We always recommend to use the newest sync client with the latest server release.
```
Comment (HanaGemela) [inline modules/admin_manual/pages/installation/system_requirements.adoc]:
```suggestion
Latest stable client supports the platforms listed below:
```

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 2]
```diff
@@ -92,9 +94,9 @@ For Linux distributions, we support, if technically feasible, the latest two ver
 * Chrome 66+
 * Safari 10+
 
-=== Sync Clients and Mobile Apps 
+=== Mobile Apps
 
-We always recommend to use the newest clients and mobile apps with the latest server release.
+We always recommend to use the newest mobile apps with the latest server release.
```
Comment (HanaGemela) [inline modules/admin_manual/pages/installation/system_requirements.adoc]:
```suggestion
Latest stable mobile apps support the platforms listed below:
```

File: modules/admin_manual/pages/installation/system_requirements.adoc (hunk 1 above)
Comment (phil-davis) [reply modules/admin_manual/pages/installation/system_requirements.adoc]: done

File: modules/admin_manual/pages/installation/system_requirements.adoc (hunk 2 above)
Comment (phil-davis) [reply modules/admin_manual/pages/installation/system_requirements.adoc]: Done

**Comment by HanaGemela** [comment]
`For Linux distributions, we support, if technically feasible, the latest two versions per platform and the previous LTS Version.`

This is in server section now. It rather belongs into desktop

**Comment by HanaGemela** [comment]
``` 
Tip | You can find out more in the changelog.
```
Not sure what is the purpose of this general link to changelog in Mobile. We can either have separate section for changelog or have desktop changelog in Desktop, mobile changelog in Mobile etc

**Comment by phil-davis** [comment]
I am just trying to get "obvious" bits of this improved.
For a whole review of what is in this `adoc` and why, maybe @settermjd you can take over co-ordinating. (I have plenty of other things in my to-do list, and I really am not anyway tasked with sorting out general documentation re-organising)

**Comment by voroyam** [comment]
> I am just trying to get "obvious" bits of this improved.

I know your pain :)

You try to help and improve one thing, and then people pile of a bunch of stuff they found wrong on the document you are editing. Then you have to implement all those things by yourself, which is not the task you head out to do. Many get discouraged or frustrated with this things and never touch docs again.

Thank you for taking the time and effort to improve the docs.

Approved.

**Comment by voroyam** [comment]
Where does this need to be backported to?

**Comment by phil-davis** [comment]
> Where does this need to be backported to?

I cherry-picked this back in to #2340 (10.4 PR) and #2341 (10.3  PR). That will get everthing back in sync.

**Comment by phil-davis** [comment]
Now the next person can suggest whatever other changes they want in another PR...

**Pull request merged**